pkg/storage/couchdb/couchdbstore.go

```diff
@@ -312,6 +314,26 @@ func (c *CouchDBStore) Get(k string) ([]byte, error) {
 	return value, nil
 }
 
+// GetBulk fetches the values associated with the given keys. This method works in an all-or-nothing manner.
+// It returns an error if any of the keys don't exist. If even one key is missing, then no values are returned.
+// The end result is equivalent to calling Get(k,v) on each key-value pair individually in a loop,
+// but should be faster since this method minimizes REST calls as long as the values were stored as JSON instead of
+// attachments.
+// If values are stored as attachments, then there are still optimizations that could be done - see TODO #124.
+func (c *CouchDBStore) GetBulk(keys ...string) ([][]byte, error) {
```

Comment:
I had originally added a ToDo here to add pagination support... but then I realized that didn't make sense, since pagination needs to be done by the caller of this method in their code. It's up to the caller to manage the size of the `keys` slice being passed in.

(I will do this in the EDV server code, which is where this method will be consumed)